opencti-platform/opencti-front/.eslintrc.js

```diff
@@ -50,6 +50,7 @@ module.exports = {
     'custom-rules',
   ],
   rules: {
+    '@typescript-eslint/no-non-null-assertion': 'off',
```

Comment:
any good reason for that?

Reply:
Well, because I don't really like the rule as a start :D
And yes, because the types can be painful in DataTable and I need sometimes to assert that the object I'm using as been valued. But may be you have better ways

Reply:
Yeah the typing seems challenging. 
Can't you use `?` or `?? something` ?

non-null assertion does not prevent the code from being called, and judging from the typing this is possible.
I know you probably use this assertion because "in this context I'm sure it's defined", but IMO it's still bug prone.